src/app/home/home.component.html

```diff
@@ -12,20 +12,41 @@
 
     <biggive-grid
       column-count="3">
-      <biggive-basic-card
+      <biggive-basic-card *ngIf="greenMatchApplicationsOpen"
         space-below='4'
         icon-colour='brand-3'
         background-image-url='/assets/images/card-background-gmf.jpg'
-        main-title='Save the Date for the Green Match Fund'
+        main-title='Applications for Green Match Fund now open'
+        subtitle='Deadline is 17 January 2023'
+        button-url='https://www.thebiggive.org.uk/charities/s/login/'
+        button-label='Apply Now'
+        button-colour-scheme='clear-primary'
+        class="basic-card-padding">
+      </biggive-basic-card>
+      <biggive-basic-card *ngIf="! greenMatchApplicationsOpen"
```

Comment:
I couldn't quite get the else part of ngIf to work, hence the slightly clunky use of *if not* logic.